### PR TITLE
[11.0] [ADD] account_budget_template

### PR DESCRIPTION
**Pull request by dalonsod**
A budget template contains different budgetary positions and defines a periodicity
to create with a button the corresponding lines.

### CODE REVIEW

File: account_budget_template/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "account_budget",
+    ],
+    "data": [
+        "security/ir.model.access.csv",
+        "views/crossovered_budget_template_view.xml",
+        "views/crossovered_budget_view.xml",
+        "views/res_config_settings_view.xml",
+    ],
+    "installable": True,
+}
```
Comment (oihane) [inline account_budget_template/__manifest__.py]: flake 8 error:
```
account_budget_template/__manifest__.py:22:2: W292 no newline at end of file
```

**Comment by oihane** [comment]
You also have this pylint warning
```
**account_budget_template/readme/DESCRIPTION.rst:1: [W7908(missing-newline-extrafiles), ]  Missing newline
account_budget_template/readme/CONTRIBUTORS.rst:1: [W7908(missing-newline-extrafiles), ]  Missing newline**
```

**Comment by oihane** [comment]
@dalonsod  You should merge the commits you've done. Check https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Review by oihane — APPROVED** [review]
Technically reviewed LGTM

**Review by anajuaristi — APPROVED** [review]
:+1: 
LGTM

**Review by lmiguens-solvos — APPROVED** [review]
LGTM

File: account_budget_template/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Account Budget Template",
+    "version": "11.0.1.2.0",
+    "category": "Accounting",
+    "license": "AGPL-3",
+    "author": "AvanzOSC,"
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-financial-tools",
```
Comment (pedrobaeza) [inline account_budget_template/__manifest__.py]: Please change the website to the new one.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: account_budget_template/__manifest__.py (hunk 2 above)
Comment (dalonsod) [reply account_budget_template/__manifest__.py]: Done!

Comment (dalonsod) [reply account_budget_template/__manifest__.py]: @pedrobaeza is this ready to merge?

**Pull request merged**